selection_sort/python/selection_sort.py

```diff
@@ -0,0 +1,20 @@
+def selection_sort(array):
+
+    # Traverse through all array elements
+    for i in range(len(array)):
+
+        # Find the minimum element in remaining
+        # unsorted array
+        min_idx = i
+        for j in range(i+1, len(array)):
+            if array[min_idx] > array[j]:
+                min_idx = j
+
+        # Swap the found minimum element with
+        # the first element
+        array[i], array[min_idx] = array[min_idx], array[i]
+
+    return array
+
+
+print(selection_sort([6,3,5,1,8,9]))
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpc7lencx6/selection_sort/python/selection_sort.py
+++ b/tmp/tmpc7lencx6/selection_sort/python/selection_sort.py
@@ -17,4 +17,4 @@
     return array
 
 
-print(selection_sort([6,3,5,1,8,9]))
+print(selection_sort([6, 3, 5, 1, 8, 9]))
```